website/docs/docs/dbt-cloud-apis/metadata-api.md

```diff
@@ -46,12 +41,12 @@ The 2023 Metadata API roadmap outlines three main uses: discovery, quality, and
 
 To improve discovery experiences, we’ll make it easier for API users to access the latest production state of a project. This is our primary focus in the first half of 2023. 
 
-- Q1 &mdash; dbt v1.5: New model fields like owner, version, and constraints.
-- Q2 &mdash; dbt v1.6: Use public models to enable [multi-project deployments](https://github.com/dbt-labs/dbt-core/discussions/6725) and access the global lineage.
-- Q2 &mdash; dbt v1.6: New and revised model and entity endpoints to unlock Semantic Layer use cases.
-- Q2 &mdash; Pagination for manageable responses and performance when querying long lists of dbt objects.
-- Q2 &mdash; Get the DAG state based on its environment’s latest production runs instead of querying per job.
-- Q3 &mdash; Access additional information about the project from the dbt logs. For instance, get catalog info without needing to generate docs.
+- Q1 &mdash; dbt v1.5: Launch new model fields like owner, version, and constraints
```

Comment:
Oh, smart moving this to Quality, We can delete from here since public models are covered as next step